packages/@lwc/perf-benchmarks-components/rollup.config.mjs

```diff
@@ -16,8 +16,6 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 const { tmpDir, styledComponents } = generateStyledComponents();
 
-const SSR_MODE = 'asyncYield';
-
```

Comment:
Rather than redeclare this in every benchmark, or import `@lwc/shared` into the benchmark code, I'm opting to just delete it from the benchmark code entirely and let the default naturally apply. YAGNI